git-cms-init

```diff
@@ -28,10 +28,20 @@ usage () {
   exit $1
 }
 
+# get default protocol from git config
+PROTOCOL=$(git config --get cms.protocol)
```

Comment:
@kpedro88 , this is failing. Due to `-e` this command fails and exit. No idea why this was not caught be the tests. But we need a fix as soon as possible

Reply:
what about changing
```bash
# get default protocol from git config
PROTOCOL=$(git config --get cms.protocol)
HAS_PROTOCOL=$?
# fallback to default
if [ "$HAS_PROTOCOL" -ne 0 ]; then
  PROTOCOL=mixed
elif ...
```
into
```bash
# get default protocol from git config, or use "mixed" by default
PROTOCOL=$(git config --get cms.protocol || echo mixed)
if ...
```
?

Reply:
See #105 .